cmd/glusterfs/main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	gfd "github.com/gluster/gluster-csi-drivers/pkg/glusterfs"
+	"github.com/gluster/gluster-csi-drivers/pkg/glusterfs/utils"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	flag.Set("logtostderr", "true")
+}
+
+func main() {
+	flag.CommandLine.Parse([]string{})
+	var config = utils.NewConfig()
+
+	cmd := &cobra.Command{
+		Use:   "glusterfs-csi-driver",
+		Short: "GlusterFS CSI driver",
+		Run: func(cmd *cobra.Command, args []string) {
+			handle(config)
+		},
+	}
+
+	cmd.Flags().AddGoFlagSet(flag.CommandLine)
+
+	cmd.PersistentFlags().StringVar(&config.NodeID, "nodeid", "", "CSI node id")
+	cmd.MarkPersistentFlagRequired("nodeid")
+
+	cmd.PersistentFlags().StringVar(&config.Endpoint, "endpoint", "", "CSI endpoint")
+
+	cmd.PersistentFlags().StringVar(&config.RestURL, "resturl", "", "glusterd2 rest endpoint")
+
+	cmd.PersistentFlags().StringVar(&config.RestUser, "username", "glustercli", "glusterd2 user name")
```

Comment:
We don't have full fledged user management yet. I'd focus on having TLS support as the pods may not run on the same host as where `glusterd2` is running